paddle/fluid/operators/mkldnn/conv_transpose_mkldnn_op.cc

```diff
@@ -123,65 +121,23 @@ class ConvTransposeMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
     std::transform(dilations.begin(), dilations.end(), dilations.begin(),
                    [](int64_t i) { return i - 1; });
 
-    const T* input_data = input->data<T>();
-    const T* filter_data = filter->data<T>();
+    const auto src_tz = paddle::framework::vectorize(input->dims());
 
-    auto src_tz = paddle::framework::vectorize<int64_t>(input->dims());
     auto iohw_weights_tz =
         paddle::framework::vectorize<int64_t>(filter->dims());
```

Comment:
Template parameter is not necessary. The `int64_t` is already a default value.
```suggestion
        paddle::framework::vectorize(filter->dims());
```